itl/command-plugins.conf

```diff
@@ -231,6 +231,10 @@ object CheckCommand "tcp" {
 			set_if = "$tcp_ssl$"
 			description = "Use SSL for the connection."
 		}
+		"--sni" = {
+			value = "$tcp_sni$"
+			description = "SSL server_name."
```

Comment:
`server_name` looks like it references a name from some standard, but I'm not sure if it even does. I'd simply go with a space here and being a little more verbose would also be nice.

Reply:
Just FYI `server_name` is the TLS extension used for SNI, see [RFC 3546](https://www.rfc-editor.org/rfc/rfc3546#section-3.1):

> In order to provide the server name, clients MAY include an extension of type "server_name" in the (extended) client hello.

I've updated the description to be a bit more verbose.